prov/gni/src/gnix_vc.c

```diff
@@ -201,6 +207,8 @@ static void __gnix_vc_pack_conn_resp(char *sbuf,
 	memcpy(cptr, resp_smsg_attr, sizeof(gni_smsg_attr_t));
 	cptr += sizeof(gni_smsg_attr_t);
 	memcpy(cptr, resp_irq_cq_mhndl, sizeof(gni_mem_handle_t));
+	cptr += sizeof(gni_mem_handle_t);
+	memcpy(cptr, &caps, sizeof(uint64_t));
```

Comment:
I think we might want a sanity check on the length here similar to the above functions.